DOCS-9966 restart runtimes dns update3.1 (#239)
mulesoft/docs-private-cloud: modules/ROOT/pages/troubleshoot-nfs.adoc

If the mount fails, you are probably missing some NFS-required libraries. If this happens, install them by running: +
`yum install nfs-utils nfs-utils-lib`

=== "mount failed: Not Permitted"

> The error message `mount failed: exit status 32. Mounting command ... not permitted` may occurr when trying to mount with the `noresvport` option since the ports may be limited. If this happens, remove the `noresvport` option and add the `insecure` flag to the mounting command.

Reviewer: ```suggestion
The error message `mount failed: exit status 32. Mounting command ... not permitted` might occur when trying to mount using the `noresvport` option because the ports might be limited. If this happens, remove the `noresvport` option and add the `insecure` flag to the mount command.
```

Reviewer: I always use "might" (possibility) rather than "may" (permission). Also, I think "since" implies time rather than causation.

Author: excellent, thanks!
